### PR TITLE
Feature/docker compose file

### PR DESCRIPTION
**Pull request by vilherda**
# Pull Request Template

## PR Checklist

- [ ] I have run `npm test` locally and all tests are passing.
- [ ] I have added/updated tests for any new behavior.
- [ ] If this is a significant change, an issue has already been created where the problem / solution was discussed: [N/A, or add link to issue here]

## PR Description

Added explanations about how to use docker-compose file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +121,30 @@ Example of multiple slot filling used in the same intent:
...
+docker-compose up -d
+```
+
+**Stop:**
+
+```docker
+docker-compose down
+```
+
+It will start the Mongo database, [NLP.js](http://localhost:3000 "NLP.js app at localhost") application and a [Mongo Express](http://localhost:8081 "Mongo-Express at localhost") to allow inspect the collections.
```
Comment (ericzon) [inline README.md]: I think that this link should fail:
Suggestion:
[NLP.js](http://localhost:3000) "NLP.js app at localhost"

Comment (ericzon) [reply README.md]: and
[Mongo Express](http://localhost:8081) "Mongo-Express at localhost"

Comment (ericzon) [reply README.md]: My bad, these links are ok. Go to merge

Comment (vilherda) [reply README.md]: These are links to reach services executed locally with docker-compose and, due to the base-configuration, these are the correct ports. I think it's ok.

**Pull request merged**